build/raise-version/raise.sh

```diff
@@ -62,6 +62,8 @@ function raiseVersionOfOurRepos {
     "git@github.com:axonivy/inscription-client.git"
     "git@github.com:axonivy/core.git"
     "git@github.com:axonivy/thirdparty-libs.git"
+    "git@github.com:axonivy/theia-ide.git"
```

Comment:
@ivy-cst with the current settings the raise-version dry run does not work, I leave it as it is for the moment
`Error: Couldn't find any versions for "@axonivy/inscription-protocol" that matches "~20.0.0-next"`

Reply:
I see, may you could just skip the yarn install part if it is in dry run. But no idea if this is given to this script. May be talk to alex what we could do here ...